app/Http/Requests/Post/PostRequest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace App\Http\Requests\Post;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class PostRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            "title" => ["required", "string"],
+            "category_id" => ["required", "integer"],
+            "published_at" => ["nullable"],
```

Comment:
esse dado no payload precisar ser um booleano (true/false). 
```suggestion
            "published_at" => ["boolean"],
```